src/NuGet.Core/NuGet.Versioning/utility/SdkAnalysisLevelMinimums.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Versioning
```

Comment:
I'm not 100% sure where an api like this would belong, but I don't NuGet.Versioning is the right place.

Reply:
made it internal in Nuget.Commands for now. As you said, when we have more use cases in the future, we can look for better location for it if we need to or just make it public.